chunk/tags.go

```diff
@@ -136,6 +141,11 @@ func (ts *Tags) UnmarshalJSON(value []byte) error {
 		if err != nil {
 			return err
 		}
+
+		// prevent a condition where a chunk was sent before shutdown
+		// and the node was turned off before the receipt was received
+		v.Sent = v.Synced
```

Comment:
I wonder if there would be similar data inconsistencies that we could not notice, which would result in strange behaviour. This is a sign that persistence is not reliable.